packages/clerk-js/src/utils/url.ts

```diff
@@ -1,4 +1,9 @@
-import { camelToSnake, createDevOrStagingUrlCache } from '@clerk/shared';
+import {
+  camelToSnake,
+  createDevOrStagingUrlCache,
+  isCurrentDevAccountPortalOrigin,
+  isLegacyDevAccountPortalOrigin,
+} from '@clerk/shared';
```

Comment:
Let's write these imports like this:

```suggestion
import { camelToSnake } from '@clerk/shared/underscore';
import { createDevOrStagingUrlCache } from '@clerk/shared/keys';
import {
  isCurrentDevAccountPortalOrigin,
  isLegacyDevAccountPortalOrigin,
} from '@clerk/shared/url';
```